jupyterlab/environment.yaml

```diff
@@ -36,7 +36,7 @@ dependencies:
   - jupyterlab-pioneer
   - jupyter-ai
   - jupyterlab-favorites >=3.2.1
-  - jupyter-scheduler >=2.5.2,<2.6.0
+  - jupyter-scheduler >=2.7.0
```

Comment:
Maybe we also want to add a pin for `<3.0`?

Reply:
Sure, I'll handle that.
Thanks for the review @krassowski :)

Reply:
The latest version I can see on pypi is `2.7.1`, do we still want to add a pin `<3.0` @krassowski?

Reply:
I added that in [`593600f`](https://github.com/nebari-dev/nebari-docker-images/pull/168/commits/593600f4203259ea4b1bd24b0da549922273b694), I can also revert if directed just in case.